### PR TITLE
Ensure camelCase naming

### LINKED ISSUE #178: Consistent use of camelCase convention

### PR DESCRIPTION
**Pull request by Nyholm**
This will fix #178 

This PR has no BC break.

### CODE REVIEW

File: src/API/Management/Users.php [hunk 1]
```diff
@@ -6,70 +6,126 @@
...
-        $response = $this->httpClient->delete(sprintf('/users/%s', $user_id));
+        $response = $this->httpClient->delete(sprintf('/users/%s', $userId));
 
         return ResponseMediator::getContent($response);
     }
 
-    public function linkAccount($user_id, $post_identities_body)
+    /**
+     * @param string $userId
+     * @param string $postIdentitiesBody
```
Comment (Nyholm) [inline src/API/Management/Users.php]: Should this be an array?

File: src/API/Management/Users.php [hunk 2]
```diff
@@ -6,70 +6,126 @@
...
         return $this->search($params);
     }
 
-    public function search($params = [])
+    /**
+     * @param array $params
+     *
+     * @return array
+     */
+    public function search(array $params = [])
```
Comment (Nyholm) [inline src/API/Management/Users.php]: I added the `array` annotation

File: src/API/Management/Users.php [hunk 3]
```diff
@@ -6,70 +6,126 @@
...
         return ResponseMediator::getContent($response);
     }
 
-    public function getAll($params = [])
+    /**
+     * @param array $params
+     *
+     * @return array
+     */
+    public function getAll(array $params = [])
```
Comment (Nyholm) [inline src/API/Management/Users.php]: I added the `array` annotation

File: src/API/Management/Users.php [hunk 4]
```diff
@@ -6,70 +6,126 @@
...
         return ResponseMediator::getContent($response);
     }
 
-    public function create($data)
+    /**
+     * @param array $data
+     *
+     * @return array
+     */
+    public function create(array $data)
```
Comment (Nyholm) [inline src/API/Management/Users.php]: I added the `array` annotation

File: src/API/Management/Users.php [hunk 5]
```diff
@@ -6,70 +6,126 @@
...
     }
 
-    public function update($user_id, $data)
+    /**
+     * @param string $userId
+     * @param array $data
+     *
+     * @return array
+     */
+    public function update($userId, array $data)
```
Comment (Nyholm) [inline src/API/Management/Users.php]: I added the `array` annotation

File: src/API/Management/Users.php (hunk 1 above)
Comment (cocojoe) [inline src/API/Management/Users.php]: I would have thought so and if not should be @glena

Comment (Nyholm) [inline src/API/Management/Users.php]: It should. I read the docs. https://auth0.com/docs/api/management/v2#!/Users/post_identities

The PR is updated.

Comment (cocojoe) [inline src/API/Management/Users.php]: can you rename to `identities` please.

Comment (Nyholm) [inline src/API/Management/Users.php]: Done

**Comment by cocojoe** [comment]
@glena LGTM

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**